api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -388,6 +398,29 @@ private FirecloudWorkspaceId generateFirecloudWorkspaceId(String namespace, Stri
     dbWorkspace.addWorkspaceUserRole(permissions);
 
     dbWorkspace = workspaceService.getDao().save(dbWorkspace);
+
+    JSONObject demoCohort = cloudStorageService.readDemoCohort();
+    // TODO: Make this a service method to avoid duplication with CohortsController
+    Cohort dbCohort = new Cohort();
+    dbCohort.setName(demoCohort.getString("name"));
+    dbCohort.setDescription(demoCohort.getString("description"));
+    dbCohort.setCriteria(demoCohort.get("criteria").toString());
+    dbCohort.setType("AoU_Discover");
+    dbCohort.setCreator(userProvider.get());
+    dbCohort.setWorkspaceId(dbWorkspace.getWorkspaceId());
+    dbCohort.setCreationTime(now);
+    dbCohort.setLastModifiedTime(now);
+    dbCohort.setVersion(1);
+    try {
+      // TODO Make this a pre-check within a transaction?
+      dbCohort = cohortDao.save(dbCohort);
+    } catch (DataIntegrityViolationException e) {
+      // TODO The exception message doesn't show up anywhere; neither logged nor returned to the
```

Comment:
What is this comment about? Is this talking about the DataIntegrity exception? Can you just log it right here?

Reply:
I am unsure. This is directly copied over from the cohortscontroller.

Reply:
Well, if neither of us knows, please just remove it.